.github/workflows/maven-release-perform.yml

```diff
@@ -31,15 +35,24 @@ jobs:
       - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
+          distribution: 'temurin'
           java-version: 11
-      - name: Prepare release settings
+          server-id: sonatype-nexus-staging
+          server-username: SONATYPE_USER
+          server-password: SONATYPE_PWD
+          gpg-private-key: ${{ secrets.GPG_KEY }}
+          gpg-passphrase: GPG_PASSPHRASE
+      - name: Download xmlstarlet
+        run: |
+          sudo apt-get install xmlstarlet
+      - name: Setup SSH
+        uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+      - name: Run Shell Script
         env:
           SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
           SONATYPE_PWD: ${{ secrets.SONATYPE_PWD }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-          GPG_KEY: ${{ secrets.GPG_KEY }}
-        run: |
-          ./.ci/prepare-settings.sh
```

Comment:
No need to use that because `setup-java` takes care of preparing the GPG key and passphrase.

Reply:
this script is good even for local laptop to setup environment.
We like CI to do same on our behalf. All CIs have problems and one day this will be required to be done manually on local.

Reply:
So you do not want to rely on setup-java at all, but rather scripts, got it.

Reply:
I would rather preserve knowledge on how to setup and CI scripts and config usually very good textual presentation of this that is never outdated in comparison to wiki pages.

Reply:
Done.